modules/test/playwright/tests/cookies-banner-web/cookiesBannerCadmin.spec.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {isolatedLayoutTest} from '../../fixtures/isolatedLayoutTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {systemSettingsPageTest} from '../../fixtures/systemSettingsPageTest';
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+
+export const test = mergeTests(
+	isolatedLayoutTest(),
+	loginTest(),
+	systemSettingsPageTest
+);
+
+test('LPD-25440 Cookie Banner Cadmin', async ({
+	page,
+	systemSettingsPage,
+}) => {
+	await test.step('Enable Third Party Cookies', async () => {
+		await systemSettingsPage.goToSystemSetting(
+			'Cookies',
+			'Preference Handling'
+		);
+
+		const enabledButton = page.getByLabel('Enabled');
+
+		await enabledButton.waitFor({state: 'visible'});
+
+		const isChecked = await enabledButton.isChecked();
+
+		if (!isChecked) {
+			await enabledButton.click();
+		}
+
+		await expect(enabledButton).toBeChecked();
+
+		const updateButton = page.getByRole('button', {
+			name: 'Update',
+		});
+
+		const saveButton = page.getByRole('button', {
+			name: 'Save',
+		});
+
+		if (await saveButton.isVisible()) {
+			await saveButton.click();
+		}
+		else if (await updateButton.isVisible()) {
+			await updateButton.click();
+		}
+
+		await waitForSuccessAlert(page);
```

Comment:
I was about to suggest to use a .config file for this, but then I realized this snippet is similar to the one in another test file. I think we can live with this duplication for some time